middle_end/flambda2/simplify/unboxing/unboxing_epa.mli

```diff
@@ -35,3 +35,6 @@ val add_extra_params_and_args :
   Continuation_extra_params_and_args.t ->
   Unboxing_types.decision ->
   Continuation_extra_params_and_args.t
+
+val compute_extra_params_in_unspecified_order :
+  Bound_parameters.t -> Unboxing_types.decision -> Bound_parameters.t
```

Comment:
From the type and the name, it's not obvious what this function does with it's first argument. I propose to rename it to `add_extra_params_in_unspecified_order`